internal/cli/root/atlas/builder.go

```diff
@@ -218,6 +218,7 @@ func shouldCheckCredentials(cmd *cobra.Command) bool {
 		fmt.Sprintf("%s %s", atlas, "completion"), // completion commands do not require credentials
 		fmt.Sprintf("%s %s", atlas, "config"),     // user wants to set credentials
 		fmt.Sprintf("%s %s", atlas, "auth"),       // user wants to set credentials
+		fmt.Sprintf("%s %s", atlas, "login"),      // user wants to set credentials
```

Comment:
thanks for the fix can you check if mongocli has the same problem please?

Reply:
also I think you should be able to do this in the search by name since `"login"` is a terminal command

Reply:
no, it doesn’t. We are not validating the credentials at the build level but only in the atlas command